src/runtime_src/core/pcie/driver/linux/xocl/subdev/mailbox.c

```diff
@@ -1924,6 +1924,16 @@ int mailbox_request(struct platform_device *pdev, void *req, size_t reqlen,
 		return -EFAULT;
 	}
 
+	if (!force) {
```

Comment:
Could you add an 1 to 2 line comment regarding this key piece of code regarding its functionality.